tests/integration/hello_bazel/BUILD

```diff
@@ -120,3 +107,9 @@ pkg_tar(
     strip_prefix = "/",
     package_dir = "federation/sample",
 )
+
+bzl_library(
```

Comment:
I'm not sure why buildifier thinks this should be at the bottom but it is very adamant about it. I can manually bump it back to the top if you'd like.